### ISSUE TITLE
Installation bug(s) for some zork-based Chromebooks

### ISSUE THREAD

**Issue by a-hurst** [WeirdTreeThing/chromebook-linux-audio#38]
Hi there,

So after a good deal of reading through forum posts and wrestling with ALSA/Pipewire debug tools, I've finally got headphone jack detection working on my ASUS Chromebook Flip CM5 (Woomax)! In the process, I figured out why this script didn't fix it for me:

1. In the UCM config in your repo for alc5682m98 cards,  the [`acp3xalc5682m98.conf`](https://github.com/WeirdTreeThing/chromebook-ucm-conf/blob/main/picasso/acp3xalc5682m98/acp3xalc5682m98.conf) file specifies the HiFi.conf file to load as the default one in alsa-ucm-conf instead of the patched one in the same folder (`File "/AMD/acp3xalc5682m98/HiFi.conf"` instead of `File "HiFi.conf"`), so your patched config file never gets loaded.

2. alsa-ucm-conf already provides a `ucm2/conf.d/acp3xalc5682m98/acp3xalc5682m98.conf` file ( a symlink to the one in `ucm2/AMD/acp3xalc5682m98/acp3xalc5682m98.conf`) and the current install script doesn't overwrite it. As a result, even with the above fixed, the custom conf file never actually gets installed. I'm guessing it's a question of updating the install script to check for/remove the symlink or allow the copying function to support overwriting existing files.

With the modified `acp3xalc5682m98.conf` file installed correctly, headphone jack detection now works perfectly!

**Comment by WeirdTreeThing** [comment]
Thanks for pointing this out, I somehow missed this. I'm not sure why my modified UCM didn't overwrite the symlink. To be honest, the best idea would be to add jack detection to the upstream UCM, along with adding the other variant of zork UCM with the different codec.

**Comment by a-hurst** [comment]
That's probably the best long-term solution, but it'll probably have to wait until Kernel 6.6 is released: the kernel driver for this card (and several others) [just got a patch](https://lore.kernel.org/all/20230802175737.263412-5-alpernebiyasak@gmail.com/) destined for Kernel 6.6 that splits the "Headset Jack" pin into separate "Headphone Jack" and "Headset Mic Jack" pins, meaning that the current UCM conf in this repo will stop working once distros update to the new kernel (since the "Headset Jack" control will no longer exist).

Probably affects some other UCMs in this repo too that aren't upstream, so figure it would be handy to have a heads-up!

**Comment by WeirdTreeThing** [comment]
Yeah, I saw that patchset some time ago but I will wait until distros start rolling out that kernel version before I switch to the new jack naming. That patchset will probably end up affecting every UCM config in my repo.
